server/server.py

```diff
@@ -83,16 +85,50 @@ def admin():
 def connect(sid, environ):
   print("connect ", sid)
 
-@sio.on('admin_ping')
-def adminconnect(sid, environ):
-  table_data = defaultdict(list)
-
-  for client in CLIENTS:
-    tablenumber = CLIENTS[client]
-    table_data[tablenumber].append(client)
-  data_to_send = {'tableData' : table_data,'gameState' : GAME_STATE}
-  sio.emit('admin_return', data_to_send)
+@sio.on('hint_ping')
+def handle_hint(sid, data):
+  puzzle = data['puzzle']
+  table = CLIENTS[sid]
+  old_hint_count = GAME_STATE[table][puzzle]['hint_count']
+  new_hint_count = old_hint_count + 1
+  GAME_STATE[table][puzzle]['hint_count'] = new_hint_count
```

Comment:
the last three lines can be simplified to `GAME_STATE[table][puzzle]['hint_count'] += 1`